### DISCUSSION TITLE
Why are relations capitalized and singular for me, but lowercase and pluralized in the docs?

### DISCUSSION THREAD

**Discussion by dominictobias** [prisma/prisma#25094] [Q&A] [answered]
### Question

In the docs (https://www.prisma.io/docs/orm/reference/prisma-client-reference#include) there is an example:

```ts
const users = await prisma.user.findMany({
  include: {
    posts: true, // Returns all fields for all posts
    profile: true, // Returns all Profile fields
  },
})
```

However when I try to do a one-to-many with my schema:

```
model User {
  id                Int                @id @default(autoincrement())
  ...
  Passkey           Passkey[]
}

model Passkey {
  id             String  @id @unique
  user           User    @relation(fields: [userId], references: [id])
  ...
}
```

Typescript errors unless I do this:

```ts
const user = await prisma.user.findUnique({
  where: { email },
  include: {
    Passkey: true,
  },
})

const mfaSetup = user.Passkey.length > 0
```

It makes me wonder if I've done something murderously wrong? Accessing an array as a singular `user.Passkey` does feel odd..

P.S. using the latest of all projects on earth as of this post's date.

**Comment by ludralph** [comment, marked as answer]
Hi @sekoyo 👋 

In your schema, you defined the `Passkey` relation field like this:

```prisma
Passkey Passkey[]
```

To get the lowercase and plural form, you can define the relation field `Passkey` like this:

```prisma
passkeys Passkey[]
```
Once you make the change, you will need to regenerate the Prisma Client using the command `npx prisma generate`. The new relation field will be `passkeys`.

**Reply by dominictobias** [reply]
Ahh oops of course, thanks! (the dangers of not checking ChatGPT output ;))
